plugins/CoreHome/templates/_dataTableActions.twig

```diff
@@ -152,4 +134,23 @@
                 </li>
             {% endif %}
         </ul>
+
+        {% if properties.show_periods %}
+            <a class="dropdown-button dataTableAction activatePeriodsSelection"
+               href="javascript:;"
+               title="{{ 'CoreHome_ChangePeriod'|translate|e('html_attr') }}"
+               data-target="dropdownPeriods{{ randomIdForDropdown }}"
+            >
+                <div><span class="icon-calendar"></span><span class="periodName">{{ clientSideParameters.period }}</span></div>
```

Comment:
That actually always shows the english parameter value, but should be translated instead. Guess something like this might work
```suggestion
                <div><span class="icon-calendar"></span><span class="periodName">{{ properties.translations[clientSideParameters.period]|default(clientSideParameters.period) }}</span></div>
```